CHANGELOG.md

```diff
@@ -3,12 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 1.0.0dev - 2024/03/11
-
-Development release of phac-nml/speciesabundance, created with the [nf-core](https://nf-co.re/) template.
+## 2.0.0 - 2024/04/18
 
 ### `Added`
 
+- The initial release of phac-nml/speciesabundance as a Nextflow pipeline following [nf-core](https://nf-co.re/) standards.
+- This pipeline will estimate the relative abundance of sequence reads originating from different species in a sample from Illumina short-read data.
+
+### `Changed`
+
+- Migrated SpeciesAbundance to a Nextflow pipeline, with integration testing using [nf-test](https://www.nf-test.com/).
```

Comment:
Could you include here a reference to the git repos of the previous pipeline we migrated from?